### PR TITLE
Apply example_ynh

### PR DESCRIPTION
**Pull request by yalh76**
## Solution

- *Apply example_ynh*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -221,6 +211,13 @@ ynh_script_progression --message="Starting a systemd service..." --weight=1
 # Start a systemd service
 ynh_systemd_action --service_name=$app --action="start"
 
+#=================================================
+# SEND A README FOR THE ADMIN
+#=================================================
+ynh_script_progression --message="Sending a readme for the admin..."
+
+# ynh_send_readme_to_admin --app_message="../conf/msg_install" --recipients=$admin_email --type='install'
```
Comment (fflorent) [inline scripts/install:219]: Should this instruction remain if it is commented?

**Review by fflorent — APPROVED** [review]
Just leaving a minor question, otherwise LGTM

File: scripts/install (hunk 1 above)
Comment (fflorent) [reply scripts/install:219]: Sorry, this line was already there.

So thank you, I merge :)

Comment (yalh76) [reply scripts/install:219]: lol I haven't seen it ^^

**Pull request merged**